### ISSUE TITLE
bug in checking of `Levels` produces warning message

### ISSUE THREAD

**Issue by mmp3** [mlampros/KernelKnn#6]
Hi @mlampros !

The new code in `dist_kernelknn.R` and `kernelknn.R` has a bug which produces the warning message:

```
Warning messages:
1: In all(Levels) : coercing argument of type 'double' to logical
```

The specific typo is in `dist_kernelknn.R` line 68 and `kernelknn.R` line 56. The line of code in both files is:

```
 if (!all(Levels) %in% unique(y)) stop("The specified 'Levels' must match the unique 'y' labels!")
```

The parentheses around `Levels` is misplaced. It should be:

```
 if (!all(Levels %in% unique(y)))
```

**Comment by mlampros** [comment]
hi @mmp3, 
can you include a reproducible example to have a look at it

**Comment by mlampros** [comment]
closed by #7
